server/routerlicious/packages/services-core/src/database.ts

```diff
@@ -140,6 +140,14 @@ export interface ICollection<T> {
     createTTLIndex?(index: any, mongoExpireAfterSeconds?: number): Promise<void>;
 }
 
+export interface IRetryAble {
```

Comment:
>IRetryAble [](http://example.com/codeflow?start=17&length=10)

IRetryable?

Reply:
Addressed in d4556832bfd0a9c0cb5bde9bbacb337ab63974ee